### PR TITLE
Formation: convert link colors to USWDS v3

### PR DESCRIPTION
**Pull request by powellkerry**
## Description
convert link colors to USWDS v3 as follows: 
```
$color-visited:              #54278f;
$color-link-default:         $color-primary;
$color-link-visited:         #54278f;
$color-link-active:          #0b4778;
```

See ticket for color comparison: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2228

## Testing done
Local testing. Further testing will be done after this is published 

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Changes have been tested in vets-website
- [ ] Changes have been tested in IE11, if applicable
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: packages/formation/sass/base/_b-variables.scss [hunk 1]
```diff
@@ -50,6 +50,11 @@ $color-primary-alt-light:    #97d4ea;
 $color-primary-alt-lightest: $color-aqua-lightest;
 $color-primary-alt-dark:     #28a0cb;
 
+$color-visited:              #54278f;
```
Comment (powellkerry) [inline packages/formation/sass/base/_b-variables.scss:53]: Maintaining this variable for backwards compatibility

**Comment by powellkerry** [comment]
@micahchiang I updated the places where hex values were the same as those we are changing in the variables. Please double check my work :)

File: packages/formation/sass/base/_b-variables.scss [hunk 2]
```diff
@@ -124,8 +132,6 @@ $color-sidebar-nav-bg:   rgba($color-black, 0.25);
 
 $color-orange: #EB7F29;
 $color-gold-darker: #988530;
-$color-light-blue: #0071bb;
-$color-old-browser-background-start: #0071bb;
 $color-old-browser-background-end: #112e51;
```
Comment (micahchiang) [inline packages/formation/sass/base/_b-variables.scss:135]: I think `#112e51` is the old hex value for `color-blue-darkest`. This should be updated to use either `$color-blue-darket`, or its hex value, `#162e51`

**Review by micahchiang — COMMENTED** [review]
Found some outstanding instances of old hex values: 
- [dash-build-pannel](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/354b249120d954ba70cc3e73200db38e34f83e93/packages/documentation/src/scss/_dashboard.scss#L2) using old `color-blue-darkest` value
- [about-mdx.mdx](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/d17e0210d497e51242449a2a571e9e5c06837e10/packages/documentation/src/pages/about-mdx.mdx#L46) using old `color-blue-darket` hex value

**Review by rsmithadhoc — APPROVED** [review]

**Review by Andrew565 — APPROVED** [review]

**Pull request merged**